make/autoconf/spec.gmk.template

```diff
@@ -435,6 +435,9 @@ FONTCONFIG_CFLAGS := @FONTCONFIG_CFLAGS@
 CUPS_CFLAGS := @CUPS_CFLAGS@
 ALSA_LIBS := @ALSA_LIBS@
 ALSA_CFLAGS := @ALSA_CFLAGS@
+KRB5_LIBS := @KRB5_LIBS@
+KRB5_CFLAGS := @KRB5_CFLAGS@
+ENABLE_LIBLINUXKRB5 := @ENABLE_LIBLINUXKRB5@
```

Comment:
A better name would perhaps be `ENABLE_LIBKRB5_LINUX` or `ENABLE_LINUX_LIBKRB5`, maybe even just `ENABLE_LIBKRB5`?